Libraries/Utilities/Alert.js

```diff
@@ -68,10 +67,9 @@ class Alert {
     title: ?string,
     message?: ?string,
     buttons?: Buttons,
-    type?: AlertType
```

Comment:
This was for convenience on iOS. How can people pass the type / style with this change?